### DISCUSSION TITLE
How to prevent running a Task, like "deploy", at the root and deploying everything?

### DISCUSSION THREAD

**Discussion by closesimple-wl** [vercel/turborepo#6035] [Help]
I only intend the "deploy" task to run in a workspace directory (using global Turborepo installation).

**Comment by pypp** [comment]
You can filter by adding ‘—filter=your-pkg’
you can read more about it [here](https://turbo.build/repo/docs/core-concepts/monorepos/filtering)

**Reply by closesimple-wl** [reply]
Thanks for the response, but the only reason I'm trying to do this is to prevent the simple mistake of a developer unintentionally deploying everything.

So using `--filter` isn't a solution, because the problem isn't developers using the filter option unintentionally.  :)
